mails/themes/modern/components/footer.html.twig

```diff
@@ -1,3 +1,27 @@
+{#**
```

Comment:
Ok this one is tricky, all those files are generated by https://github.com/PrestaShop/mjml-theme-converter

So you should update the theme sources which are also in this repository, the tricky thing here is that the generator is based on MJML and Twig and outputs some Twig files, so we'll need to find a way to add twig comments in the output

If this is not handled in the converter tool, next time a modification needs to be done it will overwrite all your current modifications